packages/wrangler/src/d1/splitter.ts

```diff
@@ -155,7 +155,7 @@ function isDollarQuoteIdentifier(str: string) {
  * Returns true if the `str` ends with a compound statement `BEGIN` or `CASE` marker.
  */
 function isCompoundStatementStart(str: string) {
-	return /\s(BEGIN|CASE)\s$/.test(str);
+	return /\s(BEGIN|CASE)\s$/i.test(str);
```

Comment:
can we please add a test case for these changes in https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/d1/splitter.test.ts 🙏 ?

Reply:
Thought of it. Going to add them soon!

Reply:
@polvallverdu Thanks for considering this. Happy to merge once there are tests added :)